<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importKeyButton">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>20</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptTextButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt Text</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textLE">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>460</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptFileButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QListWidget" name="publicKeyList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>150</width>
      <height>530</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="privateKeyList">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>150</width>
      <height>410</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="publicLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Public Keys</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="privateLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Private Keys</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="logTable">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>460</width>
      <height>490</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>230</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
